readme.md

```diff
@@ -76,33 +76,45 @@ Here is the GA gallery of exceptionally awesome WDI projects from all over the w
 
 ### Project Feedback + Evaluation
 
-Instructors will not be grading this project. We want you to use this final project as your magnum opus
-as you apply for jobs. 
+Instructors will not be grading this project. We want you to use this final project as your magnum opus as you apply for jobs.
 
-If you would like instructor feedback after your project has been completed, please **submit a link to your Github repository as a Git issue to the [`pbj-project4-gallery`](https://github.com/ga-dc/pbj-project4-gallery) repository**. If you do so **by 6:00pm on Friday**, we will do our best to have feedback returned to you by Monday morning. If you do so afterward, please just Slack an instructor from whom you'd like feedback to let them know.
+If you would like instructor feedback after your project has been completed, please **submit a link to your Github repository as a Git issue to the [`wdi-project4-gallery`](https://github.com/ga-dc/wdi-project4-gallery) repository**. If you do so **by 6:00pm on Friday**, we will do our best to have feedback returned to you some time the following week. If you do so afterward, please just Slack an instructor from whom you'd like feedback to let them know.
 
-As always, instructors will be available for 1-1 appointments and support. Please **submit technical issues to the `pbj-project4` repository as Git issues**. To schedule an appointment, please Slack the relevant instructor.
+As always, instructors will be available for 1-1 appointments and support. Please **submit technical issues to the `project4` repository as Git issues**. To schedule an appointment, please Slack the relevant instructor.
```

Comment:
This makes it sounds like the 1-1's are optional and potentially unlimited(?), whereas the references in the schedule sound mandatory and limited to two.